harness/determined/pytorch/dsat/_dsat_search_method.py

```diff
@@ -18,11 +18,6 @@
 from determined.pytorch.dsat import _defaults, _utils
 from determined.util import merge_dicts
 
-"""
-TODOs:
-    * Make sure we don't draw the same config twice in random search.
```

Comment:
If this is solved, I'm pretty sure we have other TODOs in the code that reference this same issue elsewhere